src/stability_sdk/client.py

```diff
@@ -246,6 +251,11 @@ def generate(
         :param samples: Number of samples to generate.
         :param safety: DEPRECATED/UNUSED - Cannot be disabled.
         :param classifiers: DEPRECATED/UNUSED - Has no effect on image generation.
+        :param guidance_preset: Guidance preset to use. See generation.GuidancePreset for supported values.
+        :param guidance_cuts: Number of cuts to use for guidance.
```

Comment:
Cannot be higher than the preset selected, can only be lower.

Reply:
the cuts you mean? could you actually give more details on how the different guidance parameters can or cannot override a given preset?

Reply:
`guidance_cuts` can be overridden if override is *less* than the guidance preset's default value.